src/components/ContributorShowcase/ContributorShowcase.jsx

```diff
@@ -0,0 +1,40 @@
+import { StaticQuery, graphql } from 'gatsby';
+import React from 'react';
+import styles from './ContributorShowcase.module.css';
+
+function getContributors(data) {
+  const cont = [];
+  data.contributorsJson.contributors.forEach((item) => {
+    cont.push(
+      <div className={styles.griditem} key={item.name}>
+        <img className={styles.image} alt={item.profile} src={item.avatar_url} />
+        <a href={item.profile}>{item.name}</a>
+      </div>,
+    );
+  });
+  return cont;
+}
+
+function ContributorShowcase() {
+  return (
+    <div>
+      <h2>Contributors</h2>
+      <br />
```

Comment:
Why is this `<br />` here?
I think it looks nicer without, as it then has the same spacing as the other headlines.
See Get Involved spacing below in the screenshot.

![image](https://user-images.githubusercontent.com/6052785/69083257-aa03a280-0a39-11ea-9947-81ee686cef8a.png)


```suggestion
```

Reply:
for some reason the text blended together there on my system. might've just been a weird gatsby thing, ill take a look.

Reply:
ooooh I see, that's annoying. It worked for me, but that definitely doesn't mean anything haha. If it looks odd on your machine keep it in, no worries, we'll restructure the page via #3 anyways, so not critical. 😊